R/export_cdisc.R

```diff
@@ -109,15 +114,17 @@ export_cdisc <- function(res_nca) {
                 "AAGEU",
                 "TRT01P",
                 "TRT01A",
-                "PARAM",
-                "PARAMCD",
                 "AVAL",
                 "AVALC",
                 "AVALU")
 
-
   pp_info <- res_nca$result  %>%
     filter(is.infinite(end) | PPTESTCD == "auclast") %>%
+    merge(res_nca$data$dose$data,
+          by = unname(unlist(res_nca$data$dose$columns$groups)),
+          all.x = TRUE,
+          all.y = FALSE,
+          suffixes = c("", ".y")) %>%
```

Comment:
issue: Mixing base R and dplyr patterns can impact readability.
```suggestion
    left_join(res_nca$data$dose$data,
          by = unname(unlist(res_nca$data$dose$columns$groups)),
          suffix= c("", ".y")) %>%
```